### DISCUSSION TITLE
2.1.10

### DISCUSSION THREAD

**Discussion by farag2** [farag2/Install-Office#10] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/farag2/Office/releases/tag/2.1.10'>2.1.10</a>.</em>

**Comment by VladRoboCop** [comment]
Здравствуйте. Не могли бы вы предоставить более подробную инструкцию по установке, я не совсем понимаю, что делать с файлами?

**Reply by farag2** [reply]
Распаковываешь архив. запускаешь PowerShell из папки, где office.ps1. Дальше, как написано в инструкции, пишешь, например, `.\Download.ps1 -Branch O365ProPlusRetail -Channel Current -Components Excel, PowerPoint, Word`. Скрипт начнет качать Office. После окончания. Просто из контекстного меню запусти `Install.ps1`.

**Comment by farag2** [comment]
Конечно, могу. В группу TG зайди. Я там объясню. https://t.me/sophia_chat Но чуть позже. На работе пока.
